PyRoute/Calculation/RouteCalculation.py

```diff
@@ -124,20 +125,11 @@ def get_btn(star1, star2, distance=None):
 
     @staticmethod
     def get_passenger_btn(btn, star, neighbor):
-        rich = 1 if star.tradeCode.rich else 0
-        rich += 1 if neighbor.tradeCode.rich else 0
-        # Only apply the modifier corresponding to the highest-level capital - other_capital beats sector_capital,
-        # which in turn beats subsector_capital.  The current approach makes adding a different value for other_capital
-        # (eg 3) very easy.
-        capital = 2 if star.tradeCode.sector_capital or star.tradeCode.other_capital else 1 if \
-            star.tradeCode.subsector_capital else 0
-        capital += 2 if neighbor.tradeCode.sector_capital or neighbor.tradeCode.other_capital else 1 if \
-            neighbor.tradeCode.subsector_capital else 0
-
-        passBTN = btn + rich + capital
+        passBTN = btn + star.passenger_btn_mod + neighbor.passenger_btn_mod
         return passBTN
 
     @staticmethod
+    @functools.cache
```

Comment:
If the dict lookup for this (via the cache) is faster than the calculation, we should just create a dict directly, saves on the overhead of managing the cache dict.

Reply:
Didn't seem like this case was worth that code overhead - was a simple way to grab a little bit of performance.